project-chip/connectedhomeip
# Enable Descriptor Cluster on example apps which do not have this cluster enabled

#### Problem
This issue was reported from Schneider in pre-development of a zigbee-CHIP bridge. The Descriptor Cluster is not enabled on the example bridge app.

Per-spec, this cluster should be enabled on all type of devices. It describes an endpoint instance on the node, independently from other endpoints, but also allows composition of endpoints to conform to complex device type patterns.

* Fixes #6952 Add Descriptor Cluster implementation under the Endpoint 0 of Bridge App example.

#### Change overview
Enable Descriptor Cluster on Bridge App example apps

#### Testing
How was this tested? (at least one bullet point required)
1. Launch ZAP configuration tools: ./scripts/tools/zap/configure.sh.
2. Goto Menu and open Bridge App example's configuration files.
3. Confirm Descriptor Cluster(0x001D) is enabled under Generic.
4. Run ./scripts/tools/zap_regen_all.py to generates codes for all examples and confirm no errors.
5. Run ./gn_build.sh and confirm build succeed.
